client/pages/sections/protocols/protocol-sections.js

```diff
@@ -43,29 +43,67 @@ class ProtocolSections extends PureComponent {
       sections,
       updateItem,
       editable,
-      newComments
+      newComments,
+      readonly
     } = this.props;
 
     const severityField = sections.details.fields.find(field => field.name === 'severity');
-    const severityOption = severityField.options && severityField.options.find(option => option.value === values.severity);
+    const severityOption = (severityField.options.find(option => option.value === values.severity) || {}).label;
 
     const numberOfNewComments = Object.values(newComments)
       .reduce((total, comments) => total + (comments || []).length, 0);
 
+    const speciesDetails = values.speciesDetails && values.speciesDetails.filter(s => s.name);
+
+    const na = <em>No answer provided</em>;
```

Comment:
Come on, lets have `noAnswer` please :)

Reply:
sorry, I was taking a leaf out of @lvnhmd 's book :trollface:

I'm actually shocked I wrote that.

sorrybox.gif